src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -166,16 +170,85 @@ class VersionsSidebarContainer extends React.Component<Props, State> {
-            isLoading: false,
-            isWatermarked: false,
-            versionCount: 0,
-            versions: [],
-        });
+        const shouldShowVersions = showUpsell || showUpsellWithPicture;
+
+        if (!shouldShowVersions) {
+            this.setState({
+                error: shouldShowVersionErrorWithUpsell ? messages.versionNotAvailable : messages.versionFetchError,
+                errorTitle: shouldShowVersionErrorWithUpsell
+                    ? messages.versionAccessError
+                    : messages.versionServerError,
+                isLoading: false,
+                isWatermarked: false,
+                versionCount: 0,
+                versions: [],
+            });
+        } else {
+            const isWatermarked = false;
+            const versionLimit = Infinity;
+            const versionCount = 3;
+            const versions = [
+                {
+                    type: 'file_version',
+                    id: '28338883546',
+                    permissions: {
+                        can_delete: false,
+                        can_download: false,
+                        can_preview: false,
+                        can_upload: false,
+                    },
+                    created_at: '2021-11-29T15:01:19-08:00',
+                    modified_at: '2021-11-29T15:01:19-08:00',
+                    modified_by: currentUser,
+                    trashed_at: null,
+                    size: 1875887,
+                    uploader_display_name: 'Jessica Larsen',
+                    version_number: '1',
+                },
+                {
+                    type: 'file_version',
+                    id: '283388835467',
+                    permissions: {
+                        can_delete: false,
+                        can_download: false,
+                        can_preview: false,
+                        can_upload: false,
+                    },
+                    created_at: '2021-11-29T15:01:19-08:00',
+                    modified_by: currentUser,
+                    size: 1875887,
+                    trashed_at: null,
+                    uploader_display_name: 'Jessica Larsen',
+                    version_number: '2',
+                },
+                {
+                    type: 'file_version',
+                    id: '28338883548',
+                    permissions: {
+                        can_delete: false,
+                        can_download: false,
+                        can_preview: false,
+                        can_upload: false,
+                    },
+                    created_at: '2021-11-29T15:01:19-08:00',
+                    currentUser,
+                    size: 1875887,
+                    trashed_at: null,
+                    uploader_display_name: 'Jessica Larsen',
+                    version_number: '3',
+                },
+            ];
+            this.setState({
```

Comment:
Can we use a static background image to display the "mock" versions rather than actually rendering them?

Reply:
I'm curious the benefit here? Screenreaders aren't going to parse an image (not that they're doing it here for the most part).

Reply:
Yeah, wondering how accessible an image would be. Also thinking maybe encapsulating all this into like a dummy, static version of the version history component to render instead of the actual one? I think that would separate
 the fake data better?

Reply:
^ that's totally valid. I'm mostly just curious about the trade-off here.

Reply:
@laughnan, good question. I think the main benefit is simplicity, since a background image can be applied with a couple of lines of CSS. We have no need for non-functional versions of these components outside of this upsell experiment, so this is a lot of code for just that.